block-node/block-access/src/test/java/org/hiero/block/node/access/service/BlockAccessServicePluginTest.java

```diff
@@ -117,37 +114,34 @@ void testRequestLatestBlock() throws ParseException {
     }
 
     @Test
-    @DisplayName("Request Latest and a specific Block different from latest, should fail with READ_BLOCK_NOT_FOUND")
-    void testRequestLatestBlockDifferent() throws ParseException {
-        final long blockNumber = 1;
-        final BlockRequest request = BlockRequest.newBuilder()
-                .blockNumber(blockNumber)
-                .allowUnverified(true)
-                .retrieveLatest(true)
-                .build();
+    @DisplayName("block_number is 0 and retrieve_latest is false - should return Block 0")
+    void testBlockNumberIsMinusOneAndRetrieveLatestIsFalse() throws ParseException {
+        final BlockRequest request = BlockRequest.newBuilder().build();
         toPluginPipe.onNext(BlockRequest.PROTOBUF.toBytes(request));
         // Check we get a response
         assertEquals(1, fromPluginBytes.size());
         // parse the response
         BlockResponse response = BlockResponse.PROTOBUF.parse(fromPluginBytes.get(0));
-        // check that the status is success
-        assertEquals(Code.READ_BLOCK_NOT_FOUND, response.status());
+        // check that the status is READ_BLOCK_NOT_FOUND
+        assertEquals(Code.READ_BLOCK_SUCCESS, response.status());
+        assertEquals(0, response.block().items().getFirst().blockHeader().number());
     }
 
     @Test
-    @DisplayName("block_number is -1 and retrieve_latest is false - should return READ_BLOCK_NOT_FOUND")
-    void testBlockNumberIsMinusOneAndRetrieveLatestIsFalse() throws ParseException {
+    @DisplayName("Request Latest and a specific Block different from latest, should fail with READ_BLOCK_NOT_FOUND")
```

Comment:
Potential for clarity but could be related in follow up
```suggestion
    @DisplayName("Request Latest and any Block, should fail with READ_BLOCK_NOT_FOUND")
```